src/features/interactions-touch.js

```diff
@@ -19,7 +20,12 @@ function onMultiTouch(event, ViewerDOM, tool, value, props) {
   const pinchPointDistance = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));
   const previousPointDistance = hasPinchPointDistance(value) ? value.pinchPointDistance : pinchPointDistance;
   const svgPoint = getSVGPoint(value, (x1 + x2) / 2, (y1 + y2) / 2);
-  const distanceFactor = pinchPointDistance/previousPointDistance;
+  let distanceFactor = pinchPointDistance/previousPointDistance;
```

Comment:
I have changed this to `let` as I was modifying this variable at some point in development, but now there is no reason for this change. Although it's no big deal, I should be using `const`.